app/models/custom_attribute.rb

```diff
@@ -2,7 +2,17 @@ class CustomAttribute < ApplicationRecord
   belongs_to :resource, :polymorphic => true
   serialize :serialized_value
 
+  def value=(value)
+    self.serialized_value = value
+    self[:value] = value
+  end
+
   def stored_on_provider?
     source == "VC"
   end
+
+  def field_type
+    field_type = serialized_value.class
+    field_type ? field_type.to_s.downcase.to_sym : nil
```

Comment:
It seems like this will never return `nil` since `serialized_value.class` will never be nil/ SO, I don't think we need the inline condition here